src/components/menu/menu-item/menu-item.d.ts

```diff
@@ -8,14 +8,16 @@ export interface MenuItemProps {
   icon?: IconTypes;
   submenuDirection?: string;
   selected?: boolean;
-  submenu?: string | object;
+  submenu?: string | React.ReactNode | boolean;
```

Comment:
This is now just node or boolean in the component propTypes

Reply:
Resolved now